### PR TITLE
Don't re-add users removed between load/submit.

### PR DESCRIPTION
**Pull request by chrisboyle**
Fixes: #1208

### CODE REVIEW

File: htdocs/manage/circle/editfilters.bml [hunk 1]
```diff
@@ -87,7 +87,9 @@ _c?>
             next unless $post_key =~ /^editfriend_(groupmask|maskhi)_(\w+)$/;
 
             my $trusted_u = LJ::load_user( $2 );
-            next unless $trusted_u;
+            # User might have been removed from circle between load and
+            # submit; don't re-add.
+            next unless $trusted_u && $u->trusts($trusted_u);
```
Comment (zorkian) [inline htdocs/manage/circle/editfilters.bml]: Nit: style guide says to use space inside the parenthesis: `$u->trusts( $trusted_u );`

**Comment by zorkian** [comment]
LGTM. Can you fix the style to conform with the surrounding code?

**Comment by chrisboyle** [comment]
@zorkian Done.

**Comment by zorkian** [comment]
Than you! :)

**Pull request merged**